code/datums/components/milkable.dm

```diff
@@ -0,0 +1,67 @@
+// how cows and other simplemobs are able to be milked
+
+/datum/component/milkable
+	var/mob/producer
+	var/obj/item/reagent_containers/intstorage
```

Comment:
also please seperate these nitpicking i know but initial_storage or int_storage looks much more pleasing to look at than intstorage